test/e2e/common_test.go

```diff
@@ -508,3 +510,18 @@ func (p *PodmanTestIntegration) ImageExistsInMainStore(idOrName string) bool {
 	results.WaitWithDefaultTimeout()
 	return Expect(results.ExitCode()).To(Equal(0))
 }
+
+func (p *PodmanTestIntegration) RunHealthCheck(cid string) error {
+	exitCode := 999
+	for i := 0; i < 10; i++ {
+		hc := p.Podman([]string{"healthcheck", "run", cid})
+		hc.WaitWithDefaultTimeout()
+		exitCode = hc.ExitCode()
```

Comment:
Why the `exitCode` temporary? Why not just `if hc.ExitCode() == 0`? (I assume you originally intended to reuse it, maybe a check for 125, but absent such reuse I think it's cleaner to just remote it entirely)